### PR TITLE
Log entire struct for more informative logging

### PR DESCRIPTION
**Pull request by lock14**

### CODE REVIEW

File: pkg/review/commit_review_status.go [hunk 1]
```diff
@@ -232,7 +232,7 @@ func getApprovalStatus(request *PullRequest) string {
 // issues for commits that do not have the status GithubPRApproved.
 func processReviewStatus(ctx context.Context, fetcher BreakGlassIssueFetcher, cfg *Config, commitReviewStatus CommitReviewStatus) *CommitReviewStatus {
 	logger := logging.FromContext(ctx)
-	logger.InfoContext(ctx, "processing commitReviewStatus", "commit_review_status", commitReviewStatus.SHA)
+	logger.InfoContext(ctx, "processing commitReviewStatus", "commitReviewStatus", commitReviewStatus)
```
Comment (yolocs) [inline pkg/review/commit_review_status.go:235]: Lint apparently disagrees with you on using the snake_cases

Comment (verbanicm) [inline pkg/review/commit_review_status.go:235]: fix for lint rule

**Review by verbanicm — APPROVED** [review]

**Review by yolocs — APPROVED** [review]

File: pkg/review/commit_review_status.go (hunk 1 above)
Comment (lock14) [reply pkg/review/commit_review_status.go]: fixed.

Comment (lock14) [reply pkg/review/commit_review_status.go]: fixed.

**Pull request merged**